sys-devel/bison/bison-3.8.2-r4.ebuild

```diff
@@ -62,7 +72,8 @@ src_configure() {
 	use static && append-ldflags -static
 
 	local myeconfargs=(
-		$(use_enable nls)
+		"$(use_enable nls)"
+		"$(usex nls "--with-libtextstyle" "--without-libtextstyle")"
```

Comment:
Quotes are unnecessary, and you can use the use_with function for the latter flag.
```suggestion
		$(use_enable nls)
		$(use_with nls libtextstyle)
```